### PR TITLE
Refactoring EntityFieldsOpsTest class name

### PR DESCRIPTION
**Pull request by Stan575**
EntityFieldsOpsTest class name change to match entity name in the app

### CODE REVIEW

**Review by SergeiDemyanenko — CHANGES_REQUESTED** [review]
Раздели на 2 отдельных PR:
- рефакторинг

**Comment by Stan575** [comment]
Removed editing .gitignore from this PR

**Comment by Stan575** [comment]
Editing .gitignore PR: https://github.com/SergeiDemyanenko/PlatformaticaQA/pull/233

**Comment by Stan575** [comment]
> Раздели на 2 отдельных PR:
> 
> * рефакторинг
> * редактирование .gitignore

Separated changes, PR for gitignore: https://github.com/SergeiDemyanenko/PlatformaticaQA/pull/235

**Comment by Stan575** [comment]
There are problems in code, no review needed till test fixed.

File: src/test/java/EntityFieldsOpsTest.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        getPaginationLastPage();
+        deleteLastRecord();
+    }
+
+    private WebDriverWait getWait(int timeoutSecond) {
+        return new WebDriverWait(getDriver(), timeoutSecond);
+    }
+
+    private void setup() {
+        ProjectUtils.goAndLogin(getDriver());
```
Comment (SergeiDemyanenko) [inline src/test/java/EntityFieldsOpsTest.java:159]: Заменить эту строку на ProjectUtils.loginProcedure(driver);

**Review by SergeiDemyanenko — CHANGES_REQUESTED** [review]

**Comment by Stan575** [comment]
EntityFieldOpsTest.fieldOpsView needs complete refactoring. 
[Task in Trello](https://trello.com/c/YbjNjFxf/238-fieldopsview-test-error)
Closing this PR

**Pull request closed without merging**